hack/telepresence.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Function to check if a command exists
+command_exists() {
+  command -v "$1" >/dev/null 2>&1
+}
+
+# Default environment file path
+env_file="./eventing-controller.env"
+
+# Parse command line arguments
+while [ "$#" -gt 0 ]; do
+  case "$1" in
+    -n|--namespace) namespace="$2"; shift 2;;
+    -s|--service) controller="$2"; shift 2;;
+    -p|--port) port="$2"; shift 2;;
+    -env|--env-file) env_file="$2"; shift 2;;
+    *) echo "Unknown option: $1"; exit 1;;
+  esac
+done
+
+# Check for required parameters
+if [ -z "$namespace" ]; then
+  echo "Error: Namespace parameter is missing."
+  exit 1
+fi
+
+if [ -z "$controller" ]; then
+  echo "Error: Controller parameter is missing."
```

Comment:
Currently the error message is not developer friendly. Please also tell the user what is the name of the flag they are missing.